src/styles/_nav.scss

```
@@ -0,0 +1,37 @@
+.nav {
+  color: black;
+  text-transform: uppercase;
+
+  &__list {
+    display: flex;
+    justify-content: center;
+    list-style: none;
+  }
+
+  &__item {
+    padding: 7px 20px;
+    margin: 0 22.5px;
```

избавляйся от магических чисел
```suggestion
    margin: 0 22px;
```

---

Магию убрал)